Makefile

```diff
@@ -10,14 +11,24 @@ LIB_VERSION = $(shell cat $(ROOT_DIR)/code/gradle.properties | grep "moduleVersi
 SOURCE_FILE_DIR =  $(ROOT_DIR)/code/$(PROJECT_NAME)
 AAR_FILE_DIR =  $(ROOT_DIR)/code/$(PROJECT_NAME)/build/outputs/aar
 
-create-ci: clean
-	(mkdir -p ci)
+init:
+	git config core.hooksPath .githooks
+
+format:
+	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) spotlessApply)
+	(./code/gradlew -p code/$(TEST-APP-FOLDER-NAME) spotlessApply)
+
+format-check:
+	(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) spotlessCheck)
```

Comment:
I set the make targets to apply formatting to both the source code and test app, but only to check the formatting on the source code. I can add a check to the test app code as well, but I was thinking we wouldn't want to fail a build if the test app source wasn't formatted correctly. Thoughts?